frontend/package.json

```diff
@@ -13,12 +13,14 @@
     "@xstate/fsm": "^1.6.2",
     "axios": "^0.22.0",
     "broadcastchannel-polyfill": "^1.0.1",
+    "chromium": "^3.0.3",
```

Comment:
Let's move `chromium` and `playwright` to `devDependencies` to keep dependencies organized!

Also, looks like this PR makes a lot of changes to `yarn.lock`. We mostly use yarn to manage dependencies, if we want to switch to plain `npm` we can address outside of the scope of this PR. Could you delete `package-lock.json` and run `yarn` to update the yarn lockfile instead?

Reply:
will do 🔨